blocks/library/embed/index.js

```diff
@@ -96,6 +108,19 @@ registerBlockType( 'core/embed', {
 			this.unmounting = true;
 		}
 
+		isPreviewBlacklisted( host ) {
+			if ( ! host ) {
+				return false;
+			}
+			host = host.replace( 'www.', '' );
+			for ( let i = 0; i < this.noPreview.length; i++ ) {
```

Comment:
oEmbed allows the user to embed with and without the www, even though facebook puts the www back in the embed html when its scripts run. So we need to account for that here.